tendrl/gluster_integration/manager/__init__.py

```diff
@@ -34,10 +34,12 @@ def main():
     NS.state_sync_thread = sds_sync.GlusterIntegrationSdsSyncStateThread()
 
     NS.node_context.save()
-    NS.tendrl_context.save()
-    NS.gluster_integration.definitions.save()
-    NS.gluster_integration.config.save()
+    NS.gluster.tendrl_context.save()
```

Comment:
`NS.tendrl_context` doesnt contain `integration_id` populated if just `tendrl-gluster-integration` started.
The field `NS.tendrl_context.integration_id` is populated only if import/create cluster flow is invoked. By any chance if services go down and re-started, there is no way integration_id would be populated. This would cause issues while syncing the cluster details and would add details to etc in the form `clusters//Volumes`. This would end up data in wrong state.